enclave-runtime/src/tests.rs

```diff
@@ -207,12 +221,22 @@ fn test_differentiate_getter_and_call_works() {
 			.sign(&sender.clone().into(), 0, &mrenclave, &shard);
 
 	// when
-	submit_and_execute_top(&rpc_author, &signed_getter.clone().into(), &shielding_key, shard)
-		.unwrap();
-	submit_and_execute_top(&rpc_author, &direct_top(signed_call.clone()), &shielding_key, shard)
-		.unwrap();
+	submit_operation_to_top_pool(
```

Comment:
minor re-naming, I realized `submit_and_execute_top` in fact only submits to the top pool, does not execute.